tests/testutils/ctx.go

```diff
@@ -174,7 +174,12 @@ func (ctx *RktRunCtx) RunGC() {
 		"gc",
 		"--grace-period=0s",
 	)
-	if err := exec.Command(ctx.rktBin(), rktArgs...).Run(); err != nil {
+
+	cmd := exec.Command(ctx.rktBin(), rktArgs...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
@iaguis My mistake. @jonboulle Honestly, the motivation was semaphore CI where I can't see the logs. But logging it is also an idea.